packages/ibm-products/src/components/EditInPlace/EditInPlace.tsx

```diff
@@ -159,7 +165,7 @@ export let EditInPlace = forwardRef<HTMLDivElement, EditInplaceProps>(
     const escaping = useRef(false);
 
     const tipAlignIsObject = typeof tooltipAlignment === 'object';
-    const tipAlignments: { [key: string]: string } = [
+    const tipAlignments: { [key: string]: any } = [
```

Comment:
We shouldn't use `any` for types, is there a reason why `string` doesn't work?

Reply:
it was throwing errors when it was string. 
<img width="970" alt="Screenshot 2024-06-14 at 2 35 23 PM" src="https://github.com/carbon-design-system/ibm-products/assets/10100397/1925c790-d2d5-4491-84cd-78dae5bb2924">

Reply:
```suggestion
    const tipAlignments: { [key: string]: AlignPropType } = [
```

would this work?

Reply:
I will check

Reply:
fixed, thanks!